tests/quantization/torchao/test_torchao.py

```diff
@@ -0,0 +1,444 @@
+# coding=utf-8
+# Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+import unittest
+from typing import List
+
+import numpy as np
+from transformers import AutoTokenizer, CLIPTextConfig, CLIPTextModel, CLIPTokenizer, T5EncoderModel
+
+from diffusers import (
+    AutoencoderKL,
+    FlowMatchEulerDiscreteScheduler,
+    FluxPipeline,
+    FluxTransformer2DModel,
+    TorchAoConfig,
+)
+from diffusers.utils.testing_utils import (
+    is_torch_available,
+    is_torchao_available,
+    require_torch,
+    require_torch_gpu,
+    require_torch_multi_gpu,
+    require_torchao_version_greater,
+    torch_device,
+)
+
+
+if is_torch_available():
+    import torch
+
+if is_torchao_available():
+    from torchao.dtypes import AffineQuantizedTensor
+    from torchao.dtypes.affine_quantized_tensor import TensorCoreTiledLayoutType
```

Comment:
this is renamed to `TensorCoreTiledLayout` in most recent torchao btw

Reply:
Ah got it, will update

Reply:
oh I see, yeah maybe it's modified after 0.6.1 release, we are going to do a release early next month.

Reply:
We cannot really rely on things that are not in the latest stable release, @jerryzh168 as this is a library integration. So, let's perhaps move to what is there in the latest stable?

Reply:
yeah that sounds good